Client/Frontend/UIConstants.swift

```diff
@@ -53,3 +53,12 @@ public struct UIConstants {
     /// JPEG compression quality for persisted screenshots. Must be between 0-1.
     static let ScreenshotQuality: Float = 0.3
 }
+
+private struct TempStrings {
+    // Bug 1196227 - (pbmode) [Meta] Private Browsing
+    let openInNewPrivateTab = NSLocalizedString("Open In New Private Tab", tableName: "PrivateBrowsing", comment: "Context menu option for opening a link in a new private tab")
+    let closePrivateTabsSettingsOption = NSLocalizedString("Close Private Tabs", tableName: "PrivateBrowsing", comment: "Setting for closing private tabs")
+    let whenLeavingOption = NSLocalizedString("When Leaving Private Browsing", tableName: "PrivateBrowsing", comment: "Setting for when the user leaves private browsing mode")
```

Comment:
Based on the [image](https://bugzilla.mozilla.org/attachment.cgi?id=8662568): `Will be displayed in Settings under 'Close Private Tabs'`